diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/KotlinParser.kt

```diff
@@ -50,7 +50,7 @@ class KotlinParser {
             }
         }  // I don't really understand what's going on here, but thanks to this, you can use this node in the future
         val project = KotlinCoreEnvironment.createForProduction(
-            Disposable {},
+            Disposable { },
```

Comment:
@petertrr I guess that's the case you were talking about. If the body is empty, then the `WHITE_SPACE` comes before `BLOCK` and not before `RBRACE`. Added test on this case.